tests/TaskLineRenderer.test.ts

```diff
@@ -54,21 +69,27 @@ describe('task line rendering', () => {
         // Check what we have one child, which is the rendered child
         expect(parentRender.children.length).toEqual(1);
         const li = parentRender.children[0];
-
```

Comment:
I'll review the tests later and maybe make some changes like this, and make test failures easier to understand.

But right now, my priority is in getting this PR merged, and the tests do pass, and when I break the code they do fail.

So that's good enough for me.

Feel to resolve this conversation when you've read this comment. 🤣